src/pages/my_page/user_profile/MyResults.tsx

```diff
@@ -0,0 +1,68 @@
+import { useState } from 'react';
+import { useTranslation } from 'react-i18next';
+import { useSelector } from 'react-redux';
+import { Box, CircularProgress, TablePagination, Typography } from '@mui/material';
+import { RootState } from '../../../redux/store';
+import { ContributorFieldNames, SpecificContributorFieldNames } from '../../../types/publicationFieldNames';
+import { ROWS_PER_PAGE_OPTIONS } from '../../../utils/constants';
+import { useSearchRegistrations } from '../../../utils/hooks/useSearchRegistrations';
+import { ExpressionStatement } from '../../../utils/searchHelpers';
+import { RegistrationSearchResults } from '../../search/registration_search/RegistrationSearchResults';
+
+export const MyResults = () => {
+  const { t } = useTranslation();
+
+  const personId = useSelector((store: RootState) => store.user?.cristinId) ?? '';
+
+  const [page, setPage] = useState(0);
+  const [rowsPerPage, setRowsPerPage] = useState(ROWS_PER_PAGE_OPTIONS[0]);
+
+  const [registrations, isLoadingRegistrations] = useSearchRegistrations(
+    {
+      properties: [
+        {
+          fieldName: `${ContributorFieldNames.Contributors}.${SpecificContributorFieldNames.Id}`,
+          value: personId,
+          operator: ExpressionStatement.Contains,
+        },
+      ],
+    },
+    rowsPerPage,
+    rowsPerPage * page
+  );
+
+  return (
+    <div>
+      <Typography id="registration-label" variant="h2" gutterBottom>
+        {t('my_page.my_profile.my_research_results')}
+      </Typography>
+      {isLoadingRegistrations ? (
+        <Box sx={{ display: 'flex', justifyContent: 'center', alignItems: 'center' }}>
+          <CircularProgress />
```

Comment:
Mener denne var med i stad, men forsvant kanskje med oppdateringer 😅 
```suggestion
          <CircularProgress aria-labelledby="registration-label" />
```